apps/i18n/dance/en_us.json

```diff
@@ -27,6 +18,15 @@
   "danceFeedbackUseSetSize": "Use the `set backup_dancer2 size` block to make that dancer smaller.",
   "danceFeedbackUseSetTint": "Use the `set tint` block to change a dancer's tint.",
   "danceFeedbackUseStartMapping": "Try adding the <xml><block type=\"Dancelab_startMapping\"><title name=\"SPRITE\">right_unicorn</title><title name=\"PROPERTY\">\"scale\"</title><title name=\"RANGE\">\"bass\"</title></block></xml> block to your program.",
-  "danceFeedbackKeepCoding": "Keep coding! Something's not quite right yet.",
-  "measure": "Measure:"
+  "measure": "Measure:",
+  "DanceAiNewMove": "⚠️ You need to add a block to make the dancer do a new move.",
+  "danceAIFeedbackNeedNewDancer": "⚠️ You need to make a dancer.",
+  "DanceAiFeedbackAiBlockNoParameters": "⚠️ Choose 3 emojis for AI to generate effects.",
```

Comment:
The modal uses "three emojis" rather than "3 emojis".  It seems like we should be consistent.  (The ultimate goal is make it feel like one person wrote all the strings in all places.)

Also, "AI" should be "A.I." which is the bot's identifier, the same as we now use in the modal.

Reply:
fixed